samples/sample_plugins/rotate_opencl/include/opencl_filter.h

```diff
@@ -26,7 +26,12 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 #include "mfxvideo++.h"
 #include "logger.h"
 
+#if !defined(_WIN32) && !defined(_WIN64)
```

Comment:
We can use such construction if we have some code for Linux only, but coupled with else this looks terrible. Can you, please, use "#if defined(_WIN32) || defined(_WIN64)" here?

Also, can you clarify why we need to differentiate here between Windows/Linux?

Reply:
Use _normal_ layout here, please. What you have provides useless difficulty.

Reply:
seems *normal* for me